pkg/skaffold/render/generate/generate.go

```diff
@@ -148,3 +187,21 @@ func isKustomizeDir(path string) (string, bool) {
 	}
 	return "", false
 }
+
+func isKptDir(path string) (string, bool) {
```

Comment:
nit: seems like this function could live in the `kptfile` package as a utility

Reply:
oh, the kptfile package is actually copy-pasted from the kpt repo (and can be updated as a whole in the future), so it's better not to change it for now. I can put the helper functions 
 like isKptDir in another utils sub-package when we have more helper functions.

Reply:
ahh ok got it, if we'll be importing it wholesale in the future then i guess let's leave this untouched for now